main.tex

```diff
@@ -180,6 +181,117 @@ \section*{Introduction}
+%
+The \emph{M. domestica Vgsc} sequence (EMBL accession X96668 - \cite{Williamson1996}) was aligned with the \emph{An. gambiae} AGAP004707-RA sequence (AgamP4.4 gene-set), using the Mega v7 software package \cite{Kumar2016}.
+%
+A map of equivalent codon numbers between the two species can be download from the MalariaGEN website (@@include as supplementary data file?)- 
+\url{https://www.malariagen.net/sites/default/files/content/blogs/domestica_gambiae_map.txt}.
+%%
+
+%
+Haplotypes for each chromosome of each sample were estimated (phased) using using phase informative reads (PIRs) and SHAPEIT2 v2.r837 \cite{Delaneau2013}, see Ag1000g Consortium (\cite{Ag1000gConsortium2017}) supplementary text for more details.
+%
+The SHAPEIT2 algorithm is unable to phase multi-allelic positions, therefore the two multi-allelic non-synonymous SNPs within the \emph{Vgsc} gene (>5\% ALT frequency in at least one population), altering codons \texttt{V402} and \texttt{M420}, were phased onto the haplotypes using MVNcall v1.0 \cite{Menelaou2013}.
+%
+Conservative filtering had removed one of the three known insecticide resistance conferring kdr variants, \texttt{N1570Y} \cite{Jones2012}.
+%
+After manual inspection of the read alignment revealed that the SNP call could be confidently made, it was added back into the data set and then also phased onto the haplotypes using MVNcall (see \cite{Ag1000gConsortium2017}).
+%
+To evaluate the linkage disequilibrium (LD) of non-synonymous \emph{Vgsc} mutations with the two most widespread \emph{kdr} resistance mutations (\texttt{L995S/F}), the D1 statistic was calculated using haplotypes and the python library Scikit-allel \cite{Miles2016}.
+%%
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Haplogroup networks}
+
+%
+Discerning the relationships between similar haplotypes can be difficult when using bifurcating trees as, inherently, the distance between the leaves at the tips (haplotypes) will be small.
+%
+As these relationships may be informative of the history of selection, we utilised a network approach to elucidate them.
+%
+Due to large genetic differences between haplogroups, the \texttt{L995S} and \texttt{L995F} bearing haplotypes were analysed separately; haplotypes (variants from 2L:2358158-2431617) were networked using the median-joining algorithm, designed to be used specifically with interspecific data with small genetic distances \cite{Bandelt1999}, implemented in a custom Python script.
+%
+A maximum edge (genetic) distance of two variants was employed and all networks composed of more than two haplotypes are shown.
+%
+Networks were visualised with the Python library graphviz with non-synonymous edges highlighted using the SnpEff “missense” annotations v4.1b \cite{Cingolani2012} and composite figures constructed using Inkscape.
+%%
```

Comment:
I think I would be a bit more terse here, just reporting methods and not justifying them (already done in the main text). E.g., this whole paragraph could just be something like:

"We constructed haplotype networks using the median-joining algorithm \cite{Bandelt{1999} as implemented in a custom Python script available from https://github.com/malariagen/agam-vgsc-report.

Networks were rendered with the graphviz library and a composite figure constructed using Inkscape."

The information about maximum distance and showing networks composied of more than two haplotypes would probably go better in the caption for the networks figure.